TuneUp/TuneUpWindow.xaml

```diff
@@ -78,14 +78,14 @@
             FontSize="11"
             VerticalAlignment="Center"
             SelectionUnit="FullRow"
-            SelectionMode="Extended"
+            SelectionMode="Single"
```

Comment:
@ScottAMitchell Is this the difference between multiple selection and single selection?

Reply:
Yes, I limited it to single selection to make the `centerViewOnElement` ux consistent